src/main/resources/l10n/JabRef_de.properties

```diff
@@ -2457,18 +2457,19 @@ You_must_enter_at_least_one_field_name=Sie_müssen_mindestens_einen_Feldnamen_an
 
 Non-ASCII_encoded_character_found=Nicht_ASCII-kodiertes_Zeichen_gefunden
 
-Toggle_web_search_interface=
-Background_color_for_resolved_fields=
-Color_code_for_resolved_fields=
-
-%0_files_found=
-%0_of_%1=
-One_file_found=
-The_import_finished_with_warnings\:=
-There_was_one_file_that_could_not_be_imported.=
-There_were_%0_files_which_could_not_be_imported.=
+Toggle_web_search_interface=Websuche_ein-/ausschalten
+Background_color_for_resolved_fields=Hintergrundfarbe_für_referenzierte_Felder
+Color_code_for_resolved_fields=Farbanzeige_für_referenzierte_Felder
+
+%0_files_found=%0_Dateien_gefunden
+%0_of_%1=%0_von_%1
+One_file_found=Eine_Datei_gefunden
+The_import_finished_with_warnings\:=Der Import_wurde_mit_Warnungen_abgeschlossen\:
+There_was_one_file_that_could_not_be_imported.=Eine_Datei_konnte_nicht_importiert_werden.
+There_were_%0_files_which_could_not_be_imported.=%0_Dateien_konnten_nicht_importiert_werden.
+
 Migration_help_information=Hilfe_zur_Migration
 Entered_database_has_obsolete_structure_and_is_no_longer_supported.=Die_eingegebene_Datenbank_ist_veraltet_und_wird_nicht_mehr_unterstützt.
```

Comment:
Not added by you, but I would translate a bit more lax here: "Die_gewählte_Datenbank_nutzt_eine_veraltete,_nicht_mehr_unterstützte_Struktur."